spec/features/user_can_see_random_restaurant_info_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe 'As a user when I visit dashboard' do
+  describe 'I can click shake it up link with params of price and distance' do
+    before :each do
+      visit root_path
+      stub_omniauth
+      click_link 'Sign in with Google'
+
+      expect(current_path).to eq(dashboard_path)
+      expect(page).to have_content('Happy')
+      # expect(page).to have_content('Gilmore')
```

Comment:
Removed the last name from the specs because we decided to only render the first name of the user on the dashboard page.